editor/gui/editor_file_dialog.h

```diff
@@ -282,6 +283,8 @@ class EditorFileDialog : public ConfirmationDialog {
 	void set_previews_enabled(bool p_enabled);
 	bool are_previews_enabled();
 
+	void add_side_menu(const String &p_title, Control *p_menu, real_t p_stretch_ratio = 0.5f);
```

Comment:
I think that the only required argument here should be the added control, the title should come second and be optional.